File: 4th Semester/Software Systems Engineering/Proiect/src/main/java/domain/Utilizator.java
package domain;

import javax.persistence.MappedSuperclass;

@MappedSuperclass
public class Utilizator extends Entity<Integer>{
    private String user;
    private String parola;
    private String nume;

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public Utilizator(String user, String parola, String nume) {
        this.user = user;
        this.parola = parola;
        this.nume = nume;
    }

    public Utilizator(String user, String parola) {
        this.user = user;
        this.parola = parola;
    }

    public Utilizator() {

    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getParola() {
        return parola;
    }

    public void setParola(String parola) {
        this.parola = parola;
    }

    @Override
    public String toString() {
        return "Utilizator{" +
                "user='" + user + '\'' +
                ", parola='" + parola + '\'' +
                ", nume='" + nume + '\'' +
                '}';
    }
}

File: 4th Semester/Design and programming environments/Proiect C-Sharp Client-Server Sockets/AppModel/Utilizator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppModel
{
    public class Utilizator : Entity<int>
    {
        public Utilizator(string u, string p)
        {
            user = u;
            parola = p;
        }

        public string user { get; set; }

        public string parola { get; set; }

        public override string ToString()
        {
            return user + " " + parola;
        }
    }
}
